scripts/imports/carrefour/run_carrefour_import.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# do not continue on failure
+set -e
+
+# load utils
+. scripts/imports/imports_utils.sh
+
+# this script must be launched from server root (/srv/off-pro)
+export PERL5LIB=lib:$PERL5LIB
+
+# load paths
+. <(perl -e 'use ProductOpener::Paths qw/:all/; print base_paths_loading_script()')
+
+if [[ -z "$OFF_SFTP_HOME_DIR" ]]
+then
+    >&2 "OFF_SFTP_HOME_DIR not defined, exiting"
+fi
+
+DATA_TMP_DIR=$OFF_CACHE_TMP_DIR/carrefour-data
+IMAGES_TMP_DIR=$OFF_CACHE_TMP_DIR/carrefour-images
```

Comment:
you defined it but you don't use it then !
```suggestion
```

Reply:
I'm now using it.